core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -96,6 +98,7 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
    * Splits the environment, providing the first part to this effect and the
    * second part to that effect.
    */
+  @deprecated("the Arrow combinators are deprecated", "2.0.0")
   final def ***[R1, E1 >: E, B](that: ZIO[R1, E1, B]): ZIO[(R, R1), E1, (A, B)] =
```

Comment:
I think fine to get rid of the arrow operators. Maybe just delete becauseI don't think these are being used and there isn't really a direct alternative.